cmd/acra-rotate/dbRotation.go

```diff
@@ -152,43 +79,36 @@ func rotateDb(selectQuery, updateQuery string, db *sql.DB, keystore keystore.Key
 			return false
 		}
 		logger := log.WithFields(log.Fields{"ZoneId": string(acraStructID)})
-		logger.Infof("Rotate AcraStruct with ZoneId=%s", string(acraStructID))
+		logger.Infof("Rotate AcraStruct")
```

Comment:
why we don't add zone id anymore?

Reply:
I think we should log smth like `Rotating in dry-run mode` if `dry-run` is `true`

Reply:
> why we don't log zone id anymore?
because row above create logger with zone as field that always will be included in log messages

> I think we should log smth like Rotating in dry-run mode if dry-run is true
agree